login/index.js

```diff
@@ -0,0 +1,72 @@
+const inputEmail = document.querySelector('#login-email');
+const inputPassword = document.querySelector('#login-password');
+const loginButton = document.querySelector('#login-button');
+
+inputEmail.addEventListener('focusout', noInput);
+inputEmail.addEventListener('focusout', emailCheck);
+inputEmail.addEventListener('keyup', enterKey);
+
+inputPassword.addEventListener('focusout', noInput);
+inputPassword.addEventListener('keyup', enterKey);
+
+loginButton.addEventListener('click', correctInput);
+
+
+// input 없을 때 에러 메세지
+function noInput(e) {
+  if (e.target.value === '') {
+
+    if (e.target.classList.contains('login-email')) {
+      e.target.nextElementSibling.textContent = '이메일을 입력해 주세요.';
+      e.target.style.border = '1px solid #FF5B56';
+    }
+    else if (e.target.classList.contains('login-password')) {
+      e.target.nextElementSibling.textContent = '비밀번호를 입력해 주세요.';
+      e.target.style.border = '1px solid #FF5B56';
+    }
+
+  }
+}
+
+
+// 이메일 유효성 확인
+function emailValidation(input) {
+  let reg_email = /^([0-9a-zA-Z_\.-]+)@([0-9a-zA-Z_-]+)(\.[0-9a-zA-Z_-]+){1,2}$/;
+
+  if (!reg_email.test(input)) {
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
test함수의 반환값이 boolean이기 때문에 아래처럼 코드를 수정할 수 있을 것 같아요.
```suggestion
  return reg_email.test(input);
```